openedx_events/learning/data.py

```diff
@@ -296,3 +296,23 @@ class ProgramCertificateData:
     status = attr.ib(type=str)
     url = attr.ib(type=str)
     certificate_available_date = attr.ib(type=datetime, default=None)
+
+
+@attr.s(frozen=True)
+class ExamAttemptData:
+    """
+    Attributes defined for the Open edX Exam downstream effects.
+
+    Arguments:
+        user_id (int): identifier of the user to which the grade override belongs
+        course_key (CourseKey): identifier of the course to which the grade override belongs
+        usage_key (UsageKey): identifier of the content that will get a grade override
+        requesting_user (UserData): user triggering event, sent only when instructor modifies an exam attempt
+        credit_requirement_status (str): status to change the student's CreditRequirement to
+    """
+
+    user_id = attr.ib(type=int)
+    course_key = attr.ib(type=CourseKey)
+    usage_key = attr.ib(type=UsageKey)
+    requesting_user = attr.ib(type=UserData, default=None)
+    credit_requirement_status = attr.ib(type=str, default=None)
```

Comment:
I thought we didn't need this anymore. Did something change?

Reply:
Nope just forgot to remove it